### PR TITLE
Continue adding docs

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: docs/guide/en/usage-with-middleware.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Usage with middleware
+
+`HydratorAttributeParametersResolver` is a 
+[custom implementation pf parameters resolver](https://github.com/yiisoft/middleware-dispatcher?tab=readme-ov-file#creating-your-own-implementation-of-parameters-resolver)
```
Comment (samdark) [inline docs/guide/en/usage-with-middleware.md:4]:
```suggestion
[custom implementation of parameters resolver](https://github.com/yiisoft/middleware-dispatcher?tab=readme-ov-file#creating-your-own-implementation-of-parameters-resolver) 
```

File: README.md [hunk 2]
```diff
@@ -35,107 +35,50 @@ composer require yiisoft/input-http
...
-Both will expect request parameters in request named same as DTO properties.
+Basic steps:
+
+- Configure [storing request](docs/guide/en/storing-request.md).
+- Configure [parameters resolver](docs/guide/en/parameters-resolvers.md).
+- Create DTO ([request input](docs/guide/en/request-input.md)).
+- Mark DTO properties with [hydrator attributes](docs/guide/en/hydrator-attributes.md) provided by this package.
+- Add DTO class name as type hint to a class method argument where you want to it to be resolved.
+
+For other available options, see the [guide](docs/en/guide)
```
Comment (vjik) [inline README.md:77]:
```suggestion
For other available options, see the [guide](docs/guide/en).
```

File: docs/guide/en/parameters-resolvers.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+$dispatcher = new MiddlewareDispatcher(
+    new MiddlewareFactory(
+        $container, 
+        new CompositeParametersResolver($existingResolver1, $existingResolver1, $addedResolver),
+    ),
+    $eventDispatcher,
+);
+```
+
+## `RequestInputParametersResolver`
```
Comment (vjik) [inline docs/guide/en/parameters-resolvers.md:122]: Add note about usage validating hydartor for request input DTO automatic validation.

**Review by vjik — CHANGES_REQUESTED** [review]

File: docs/guide/en/parameters-resolvers.md (hunk 3 above)
Comment (arogachev) [reply docs/guide/en/parameters-resolvers.md:122]: :heavy_check_mark:

**Pull request merged**